pallets/gear-bank/src/lib.rs

```diff
@@ -228,6 +233,18 @@ pub mod pallet {
     #[pallet::storage]
     type Bank<T> = StorageMap<_, Identity, AccountIdOf<T>, BankAccount<BalanceOf<T>>>;
 
+    /// The default account ID of the Gear Bank.
+    pub struct DefaulBankAddress<T: Config>(PhantomData<T>);
+    impl<T: Config> Get<AccountIdOf<T>> for DefaulBankAddress<T> {
+        fn get() -> AccountIdOf<T> {
+            Pallet::<T>::bank_address()
+        }
+    }
+
+    // Private storage to hold the GearBank's AccountId.
+    #[pallet::storage]
+    pub type BankAddress<T> = StorageValue<_, AccountIdOf<T>, ValueQuery, DefaulBankAddress<T>>;
```

Comment:
This default works only as rust types shortcut, it doesnt insert value into storage on first access, so every caller will do compute of the address: set it somewhere explicitly please